spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -391,6 +391,7 @@ class Advert < ActiveRecord::Base
 
       Advert.field_specs.delete(:c_id)
       Advert.index_definitions.delete_if { |spec| spec.fields == ["c_id"] }
+      Advert.constraint_definitions.delete_if { |spec| spec.foreign_key_column == "c_id" }
```

Comment:
Because this was missing, it was leaving a foreign key definition around, which added extra junk to the migrations below.